src/IntegratesQueryBuilder.js

```diff
@@ -0,0 +1,43 @@
+import { Mixin } from 'mixwith'
+import Javel from '@/Javel'
+
+/**
+ * @needs MakesRequests
+ */
+export default Mixin((superclass) => class extends superclass
+{
+    static query() {
+        let queryBuilderModule
+
+        try {
+            queryBuilderModule = Javel.resolveOptionalModule('js-query-builder')
+        } catch (e) {
+            throw new Error('IntegratesQueryBuilder mixin requires optional "js-query-builder" module to be registered.')
```

Comment:
I don't think the `try/catch` here is necessary anymore since the `resolveOptionalModule` method already has it's own `try/catch`.

Reply:
I've wrapped resolving of the module with `try/catch` in order to make an exception message more sensible.

Reply:
Okay makes sense. 🙂